lib/rspec/request_snapshot/matcher.rb

```diff
@@ -32,7 +32,7 @@ module Rspec::RequestSnapshot
     end
 
     def format
-      @options[:format]&.to_sym || RSpec.configuration.request_snapshots_default_format
+      (@options[:format] && @options[:format].to_sym) || RSpec.configuration.request_snapshots_default_format
```

Comment:
Use safe navigation (`&.`) instead of checking if an object exists before calling the method.

Reply:
This was removed intentionally to support older ruby versions. There is an open issue that the gem stopped working with older ruby versions